SCOPES.md

```diff
@@ -0,0 +1,57 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:beetmover`
+  * Used for Firefox/Fennec/Devedition style beetmover tasks
+
+* `project:comm:thunderbird:releng:beetmover`
+  * Used for Thunderbird style beetmover tasks
+
+
+# Scopes
+* `{scope_prefix}:action:push-to-candidates`
+  * Tells beetmoverscript to copy to candidates directory in S3 the corresponding release artifacts
+  * **Conflicts**: with any other `{scope_prefix}:action:*`
+  * **Branch Restrictions**: `all-candidates-tasks`
```

Comment:
In this case there is no restriction, I've used **Branch Restrictions** as a means of saying what scriptworker will barf on, and less about what the taskgraph will produce.

And in doing so it gets the definitions from scriptworker, specifically https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/constants.py#L244 so if its missing in that set it's `None`, or it can be `all-nightly-branches` etc.

Reply:
Okay thank you, that makes sense, setting it to None